src/main/java/com/string/ReverseWords.java

```diff
@@ -0,0 +1,29 @@
+package strings;
+
+public class ReverseWords {
+  /**
+   * Converts all of the words in this {@code String} to reversed words
+   *
+   * @param s the string to convert
+   * @return the {@code String}, converted to a string with reveresed words.
+   */
+  public static String isReverseWords(String s) {
```

Comment:
Method name is kind of not in sync with what it does, i.e. 

```java
 public static String isReverseWords(String s) {...} // Suggests that this will return boolean value. 

 public static String returnReverseWords(String s) {...} // Suggests that this will return reversed string value. 
```

**NOTE** : This is just an observation or suggestion.

Reply:
I renamed it `isReverseWords` to `returnRevereseWords`